common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -501,4 +501,15 @@ public static byte[] base64Decode(@NonNull final String input, int flags, @NonNu
             throw e;
         }
     }
+
+    /**
+     * Helper function to overwrite all characters in an array with zero
+     * @param chars
+     */
+    public static void overwriteWithZero(char[] chars) {
+        int length = chars == null ? 0: chars.length;
```

Comment:
Alternatively you can just return early from the method if chars == null

or even better just make the method NonNull

Reply:
I prefer single return statement in small methods. If I make the method non-null then the calling code in some places might get complicated.